### ISSUE TITLE
Accessing IntraACL features through API calls

### ISSUE THREAD

**Issue by centaure** [mediawiki4intranet/IntraACL#76]
Is it possible to alter permissions/ACL of a page or groups etc. programatically using API calls to IntraACL functionality? 

Or is there any other way to control permission/groups programatically? 

Any advice ? I have shell access to my mediawiki.

**Comment by vitalif** [comment]
Of course, you just need to create a wiki page named ACL:Page/YourArticleTitle with the appropriate content using mw code api
